ecma402/finished-proposals.md

```diff
@@ -13,11 +13,10 @@ Finished proposals are proposals that have reached stage 4, and are included in
 | [`Intl.ListFormat`][intl.listformat]                                                         | Zibi Braniecki                       | Zibi Braniecki                       | [July 2020][intl.listformat-notes]                                | 2021              |
 | [`DateTimeFormat` `dateStyle` & `timeStyle`][datetimeformat]                                 | Zibi Braniecki                       | Zibi Braniecki                       | [July 2020][datetimeformat-notes]                                 | 2021              |
 | [Intl.DisplayNames][proposal-intl-displaynames]                                              | Frank Tang                           | Frank Tang                           | [September 2020][proposal-intl-displaynames-notes]                | 2021              |
-| [`Intl.DateFormat.prototype.formatRange`][formatrange]                                       | Felipe Balbontín                     | Sathya Gunasekaran                   | January&nbsp;2021                                                 | 2021              |
-| [`Intl.Segmenter`: Unicode Segmentation in JavaScript][intl-segmenter]                       | Daniel Ehrenberg, Richard Gibson     | Richard Gibson                       | October&nbsp;2021                                                 | 2022              |
-| [Extend TimeZoneName Option Proposal][extend-timezonename]                                   | FrankYFTang                          | FrankYFTang                          | December&nbsp;2021                                                | 2022              |
-| [`Intl.DisplayNames v2`][intl.displaynames-v2]                                               | Frank Tang                           | Frank Tang                           | December&nbsp;2021                                                | 2022              |
-
+| [`Intl.DateFormat.prototype.formatRange`][formatrange]                                       | Felipe Balbontín                     | Sathya Gunasekaran                   | [January 2021][formatrange-notes]                                                 | 2021              |
+| [`Intl.Segmenter`: Unicode Segmentation in JavaScript][intl-segmenter]                       | Daniel Ehrenberg, Richard Gibson     | Richard Gibson                       | [October 2021][intl-segmenter-notes]                                                 | 2022              |
+| [Extend TimeZoneName Option Proposal][extend-timezonename]                                   | Frank Tang                           | Frank Tang                           | December&nbsp;2021                                                | 2022              |
```

Comment:
unifi "frank tang" name?

see https://github.com/tc39/proposals/search?q=Frank

CC @FrankYFTang

Reply:
CC @ljharb

Reply:
yes, they should all be the same; it's up to Frank what's used tho.

Reply:
https://github.com/search?q=org%3Atc39+Frank+Tang+FrankYFTang&type=Code

it seems the "frank tang" is used more